index.html

```diff
@@ -28,7 +29,7 @@
                 </ul>
             </nav>
             <div class="movil-menu">
```

Comment:
- [ ] You did a good job on the functionalities but for this project, you are supposed to stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) design as much as possible. Please reduce the weight of the close(X) button and increase the font size for mobile menu options to make sure it matches the template.

### Your design

![image](https://user-images.githubusercontent.com/83803414/167683666-f1ae5d4f-9f7e-49a2-afe3-d97878144aa7.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/167683714-afbe4650-8b28-4180-a9eb-20e188c599ff.png)